build/script.js

```diff
@@ -242,7 +242,7 @@ function patchBuildInformation() {
                   patterns: [
                     {match: 'BUILD_GIT_COMMIT', replacement: commit},
                     {match: 'BUILD_DASHBOARD_VERSION', replacement: conf.deploy.version.release},
-                    {match: 'BUILD_YEAR', replacement: new Date().getFullYear()},
+                    {match: 'BUILD_YEAR', replacement: "2018"},
```

Comment:
Instead of hard-coding the date here, instead I'd put it in build/conf.js and reference the new variable. Since this seems tied to the release/commit, look for the "version" variable and maybe add a "buildDate" attribute?

Reply:
actually, I think the 'build' part in the name can be misleading. It is meant to contain the date/year of the last source change.
Just wanted to keep the patch simple for the start.
Will push an update for using conf.js